OSDOCS-17042 [CQA] Update infrastructure machine set creation documentation (#105997)
openshift/openshift-docs: machine_management/creating-infrastructure-machinesets.adoc

toc::[]

[role="_abstract"]
You can use infrastructure machine sets to create machines that host only infrastructure components, such as the default router, the integrated container image registry, and the components for cluster metrics and monitoring. These infrastructure machines are not counted toward the total number of subscriptions that are required to run the environment.

> 

Reviewer: Did this info get preserved somewhere?

include::snippets/machine-user-provisioned-limitations.adoc[leveloffset=+1]

include::modules/infrastructure-components.adoc[leveloffset=+1]

> 

Author: Incorporated the feedback from #105995 here. Thanks

toc::[]

[role="_abstract"]

> To reduce subscription costs, you can use infrastructure machine sets to create machines that host only infrastructure components, such as the default router, the integrated container image registry,  cluster metrics, and monitoring. These infrastructure machines are not counted toward the total number of subscriptions that are required to run the environment.

Reviewer 2: extra space

```suggestion
To reduce subscription costs, you can use infrastructure machine sets to create machines that host only infrastructure components, such as the default router, the integrated container image registry, cluster metrics, and monitoring. These infrastructure machines are not counted toward the total number of subscriptions that are required to run the environment.
```

For information about infrastructure nodes and which components can run on infrastructure nodes, see the "Red Hat OpenShift control plane and infrastructure nodes" section in the OpenShift sizing and subscription guide for enterprise Kubernetes document.

> To create an infrastructure node, you can use a machine set, label the node, or use a machine config pool. Use the sample compute machine set for your cloud to deploy an infrastructure machine set. Modify the sample configuration file with the details of your environment.

Reviewer 2: Honestly, whether or not you agree with the spirit of a paragraph that's just "links to other sections in this page", I think without at least keeping the titles of these sections here this paragraph is a bit useless (as a user I might think "great, all I see until my mouse-scrolling finger cramps are a bunch of sample YAML sections, what am I looking for here?").

I would honestly suggest that in this round of DITA-migration-only fixes, you replace the xrefs with verbatim quotes of the section titles users should be looking for, and make as few other changes as you can: 
```suggestion
To create an infrastructure node, see "Creating a compute machine set", "Creating an infrastructure node", or "Creating an infrastructure node".
```

Author: Good idea on the xref replacement.
